raiden/api/rest.py

```diff
@@ -456,6 +459,7 @@ def __init__(self, raiden_api: RaidenAPI = None, rpc_client: JSONRPCClient = Non
         self.sent_success_payment_schema = EventPaymentSentSuccessSchema()
         self.received_success_payment_schema = EventPaymentReceivedSuccessSchema()
         self.failed_payment_schema = EventPaymentSentFailedSchema()
+        self.notification_schema = NotificationSchema()
```

Comment:
It depends on how much these are used. Instantiation takes some considerable time, and if this object will live in the memory for a long time, it makes sense to have an early instantiation of the schemas happening only once rather than many times on the fly.